presto-product-tests/conf/docker/singlenode-simple-impersonation/docker-compose.yml

```diff
@@ -0,0 +1,36 @@
+version: '2'
+services:
+  hadoop-master:
+    hostname: hadoop-master
+    image: 'teradatalabs/cdh5-hive:2'
+    ports:
+      - '1080:1080'
+      - '8020:8020'
+      - '8088:8088'
+      - '9083:9083'
+      - '10000:10000'
+      - '50070:50070'
+      - '50075:50075'
+
+  presto-master:
+    hostname: presto-master
+    image: 'teradatalabs/centos6-java8-oracle:jdk1.8.0_40-4e54a19'
+    depends_on:
+      - 'hadoop-master'
+    command: /docker/volumes/presto/presto-product-tests/conf/docker/files/presto-launcher-wrapper.sh singlenode run
+    volumes:
+      - ../../../../:/docker/volumes/presto
+      - ../../../../presto-product-tests/conf/presto/etc/singlenode-simple-with-impersonation.properties:/docker/volumes/presto/presto-product-tests/conf/presto/etc/catalog/hive.properties
```

Comment:
I think `singlenode-simple-with-impersonation.properties` should have `hive` word in the name. So, it is suggest that it is `hive` connector properties. You can call it, say, `hive-hdfs-impersonation.properties`. And than for kerberos you can call it `hive-kerberos-hdfs-impersonation.properties`, or `hive-kerberos-hdfs-no-impersonation.properties`. 

Than based on the fact that we have more than one additional connectors maybe it make sense to create separate directory within the `conf/presto` directory where we will keep all the properties files for environment specific connectors. We can call that catalog say `environment-specific-catalogs`.